kratos/tests/cpp_tests/modeler/test_surrogate_boundary_modeler.cpp

```diff
@@ -0,0 +1,192 @@
+//    |  /           |
```

Comment:
The indentation is a bit messy. There are also several debug prints that should be removed before merging.